hotel_reservation/models/hotel_folio.py

```diff
@@ -0,0 +1,106 @@
+# See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class HotelFolio(models.Model):
+
+    _inherit = "hotel.folio"
+    _order = "reservation_id desc"
+
+    reservation_id = fields.Many2one(
+        "hotel.reservation", "Reservation", ondelete="restrict"
+    )
+
+    def write(self, vals):
+        context = dict(self._context)
+        context.update({"from_reservation": True})
```

Comment:
Where is it used?
I don't find with_context here

Reply:
Done all suggested changes. Please review it.